packages/shared/src/WithNativeNavigation.js

```diff
@@ -18,16 +20,31 @@ function withNativeNavigation<Props: {}>(
 ): React.ComponentType<$Diff<Props, InjectedProps>> {
   return class WithNativeNavigation extends React.Component<*> {
     currentIndex: number;
+    navigator: NavigatorType;
 
     constructor(props) {
       super(props);
       this.currentIndex = 0;
     }
+
+    nestings = function nestings(nav) {
```

Comment:
could this be arrow function instead?

Reply:
I would prefer to stick with that approach, I think it's more optimal.

Reply:
> I think it's more optimal

Please elaborate 😄

Reply:
https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1

Reply:
I am still not convinced. We have chosen to go for arrow functions in this project, and we should stick to it.